### ISSUE TITLE
[Spotify Player] Grammar mistake in readme

### ISSUE THREAD

**Issue by not-first** [raycast/extensions#5822]
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

In the current README there is a small grammar mistake. Not urgent or important at all, but would only require a little change!

ERROR IN:
Has we forgotten something? Let us know and [create an issue](https://github.com/raycast/extensions/issues/new/choose)!

SHOULD BE:
Have we forgotten something? Let us know and [create an issue](https://github.com/raycast/extensions/issues/new/choose)!

### Steps To Reproduce

1. Open README for spotify player extension

### Current Behaviour

Bottom line of README states:
Has we forgotten something? Let us know and [create an issue](https://github.com/raycast/extensions/issues/new/choose)!

### Expected Behaviour

Bottom line of README should state:
Have we forgotten something? Let us know and [create an issue](https://github.com/raycast/extensions/issues/new/choose)!
